app/routes/index.js

```diff
@@ -7,14 +7,17 @@ export default Ember.Route.extend({
     model() {
         var getTotalPayload = '{"size": 0, "from": 0,"query": {"bool": {"must": {"query_string": {"query": "*"}}, "filter": [{"term": {"type.raw": "preprint"}}]}}}';
 
+        function formatNumberWithCommas(x) {
```

Comment:
Maybe not relevant here, but this looks super useful.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

The browser compatibility chart seems to suggest it works in safari, if you let it choose the locale for you. (eg it doesn't understand all the arguments, because Safari is weird)